install/configuring_inventory_file.adoc

```diff
@@ -1040,37 +1040,6 @@ openshift_hosted_registry_storage_volume_name=registry
 openshift_hosted_registry_storage_volume_size=10Gi
 ----
 
-[discrete]
-[[advanced-install-cluster-registry-storage-upgrade-nfs]]
-==== Upgrading or Installing {product-title} with NFS
-
-The use of NFS for the core {product-title} components is not recommended, as NFS
```

Comment:
Are we testing NFS with the core {product-title} components where we feel its save to remove this? 
Are we going to adjust the notes for logging, metrics, monitoring, etc? If not we need to not simply remove this but adjust this to explicitly call out where we support/recommend NFS (based on testing).

Reply:
this section of doc that is being removed is specific to the registry.  nothing has changed about the docs w/ respect to using NFS w/ logging/metrics/etc.  That content still exists (and still results in an unsupported cluster if you use it, i believe, since the ansible logic wasn't changed for those components either).

Reply:
there is at least one known issue [1] with nfs as registry storage, we should at least mention the workaround (i.e. `no_wdelay`) first [2]

1. https://access.redhat.com/documentation/en-us/openshift_container_platform/3.5/html/installation_and_configuration/setting-up-the-registry
2. https://docs.openshift.com/container-platform/3.6/install_config/registry/registry_known_issues.html#known-issue-nfs-image-push-fails